src/routes/importing-route.php

```diff
@@ -128,7 +128,7 @@ protected function get_importer( $plugin, $type ) {
 	 *
 	 * @return bool Whether or not the current user is allowed to import.
 	 */
-	public function can_import() {
+	public function is_user_permitted_to_import() {
 		return \current_user_can( 'edit_posts' );
```

Comment:
This seems to allow users with the `edit_posts` capability (which is the subscriber role and above) to deactivate any conflicting plugins. This may not be as intended.